src/openlifu/io/LIFUUart.py

```diff
@@ -0,0 +1,337 @@
+import asyncio
+import logging
+import threading
+import time
+
+import serial
+import serial.tools.list_ports
+
+from openlifu.io.config import (
+    OW_ACK,
+    OW_CMD_NOP,
+    OW_END_BYTE,
+    OW_ERROR,
+    OW_START_BYTE,
+)
+from openlifu.io.LIFUSignal import LIFUSignal
+from openlifu.io.uartpacket import UartPacket
+from openlifu.io.utils import util_crc16
+
+# Set up logging
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Setting the root logger to debug level at the top of this file is not appropriate as this will affect logging globally just when the file gets imported. I will adjust this.